src/pybkgmodel/processing.py

```diff
@@ -11,242 +14,897 @@
+        x_min : astropy.units.quantity.Quantity
+            Minimal positon along the x/azimuth axis.
+        x_max : astropy.units.quantity.Quantity
+            Maximum positon along the x/azimuth axis.
+        x_nbins : int
+            Number of bins along the x/azimuth axis.
+        y_min : astropy.units.quantity.Quantity
+            Minimal positon along the y/Zenith axis.
+        y_max : astropy.units.quantity.Quantity
+            Maximum positon along the y/Zenith axis.
+        y_nbins : int
+            Number of bins along the y/Zenith axis.
+        e_min : astropy.units.quantity.Quantity
+            Minimal energy edge of the bkg maps.
+        e_max : astropy.units.quantity.Quantity
+            Maximum energy edge of the bkg maps.
+        e_nbins : int
+            Number of bins along the energy axis
+        time_delta : astropy.units.quantity.Quantity
+            Time difference between runs for the run matching. 
+        pointing_delta : astropy.units.quantity.Quantity
+            Pointing difference between runs for run matching. 
+        """
+        
+        super().__init__(self, 
+                files = files,
+                cuts = cuts, 
+                out_dir = out_dir, 
+                out_prefix = out_prefix, 
+                overwrite = overwrite,  
+                x_min = x_min, 
+                x_max = x_max, 
+                y_min = y_min,
+                y_max = y_max, 
+                x_nbins = x_nbins, 
+                y_nbins = y_nbins,
+                e_min = e_min,
+                e_max = e_max,
+                e_nbins = e_nbins
             
-            # TODO: get all params via config
-            bkg_map = runwise_exclusion_map(
-                run,
-                runs,
-                xedges,
-                yedges,
-                energy_edges,
-                regions,
-                cuts=cuts,
-                time_delta=0.2*u.hr,
-                pointing_delta=2*u.deg
-            )
-
-            base_name = os.path.basename(run.file_name)
-            base_name, _ = os.path.splitext(base_name)
+        )
+        self.pointing_delta = pointing_delta
+        self.time_delta     = time_delta
+        self.bkg_map_maker = WobbleMap(runs=self.runs,
+                                        x_edges=self.x_edges,
+                                        y_edges=self.y_edges,
+                                        e_edges=self.e_edges,
+                                        cuts=self.cuts,
+                                        time_delta=self.time_delta,
+                                        pointing_delta=self.pointing_delta
+                                        )
+
+    @BkgMakerBase.bkg_map_maker.setter
+    def bkg_map_maker(self, maker):
+        if not isinstance(maker, WobbleMap):
+            raise TypeError(f"Maker must be of type {WobbleMap}")
+        BkgMakerBase.bkg_map_maker.fset(maker)
```

Comment:
Here and below: I think this can be simplified to` BkgMakerBase.bkg_map_maker = maker`

Reply:
I agree and changed it following your suggestion.